wandb/sdk/internal/internal.py

```diff
@@ -62,7 +62,7 @@ def wandb_internal(
 
     """
     # mark this process as internal
-    wandb._set_internal_process()
+    wandb._set_internal_process()  # type: ignore
```

Comment:
why?

Reply:
it's in __init__.py, but not in the stub, so mypy can't find it. but I don't want to put it in there. this should be moved into a separate module, but it's outside of the scope of this PR.